cfgov/legacy/housing_counselor/cleaner.py

```diff
@@ -71,9 +71,9 @@ def title_case(s):
     )
 
     parts[0] = parts[0].title()
-    parts = map(
+    parts = list(map(
         lambda part: part.title() if part not in lower_case else part,
         parts
-    )
+    ))
```

Comment:
Is this change needed as the resulting map `parts` is joined just below?

Reply:
Oh, no, I don't think so.